source/Calamari.Common/Features/StructuredVariables/JsonFormatVariableReplacer.cs

```diff
@@ -22,27 +17,22 @@ public JsonFormatVariableReplacer(ICalamariFileSystem fileSystem)
             this.fileSystem = fileSystem;
         }
 
-        public string FileFormatName => "JSON";
+        public string FileFormatName => StructuredConfigVariablesFileFormats.Json;
 
-        public bool TryModifyFile(string filePath, IVariables variables)
+        public bool IsBestReplacerForFileName(string fileName)
         {
-            JToken root;
-            try
```

Comment:
Try/catch removed because we're not doing auto detection based on parsing any more.